tests/winter_django/test_view.py

```diff
@@ -13,3 +16,19 @@ def test_create_django_urls():
 
     assert len(urls) == 1
     assert urls[0].pattern.regex.pattern == expected_url_pattern
+
+
+def test_create_django_urls_from_routes():
+    expected_url_pattern = (
+        '^with-path-parameters/(?P<param1>[^/]+)/(?P<param2>\\d+)/'
+        '(?P<param3>((one)|(two)))/(?P<param4>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/'
+        '(?P<param5>((1)|(2)))/$'
+    )
+    component = Component.get_by_cls(APIWithPathParameters)
+    method = component.get_method('test')
+    route = get_route(method)
+
+    urls = create_django_urls_from_routes(routes=[route])
+
+    assert len(urls) == 1
+    assert urls[0].pattern.regex.pattern == expected_url_pattern
```

Comment:
Also - please test the result using HTTP client, otherwise we can't be sure that everything works fine. Unit tests are not enough and probably not useful at all in this case.